django-backend/fecfiler/reports/migrations/0015_form99_filing_frequency.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 5.2 on 2025-05-13 19:21
+
+from django.db import migrations, models
+
+
+def form_99_add_default_text_code(apps, schema_editor):
+    report_model = apps.get_model("reports", "Report")  # noqa
+    reports_to_update = report_model.objects.filter(
+        models.Q(form_99__isnull=False),
```

Comment:
Do we want to add a clause here to exempt submitted reports? (maybe this is an Aurelia/Gregg question).